compiler/stable_mir/src/mir/mono.rs

```diff
@@ -90,6 +90,10 @@ impl Instance {
         with(|context| context.instance_name(self.def, true))
     }
 
+    pub fn intrinsic_name(&self) -> Option<Symbol> {
```

Comment:
Can you please document this function?

Reply:
Done.

Reply:
Can you move the check up here? Only invoke `context.intrinsic_name` if this is an intrinsic type?

Reply:
I've moved the check to be here.